RELEASE.md

```diff
@@ -0,0 +1,122 @@
+Release Process
+===============
+
+This document outlines how to create a release of prototool.
+
+1.  Set up some environment variables for use later.
+
+    ```
+    # This is the version being released.
+    VERSION=1.21.0
+
+    # This is the branch from which $VERSION will be released.
+    # This is almost always dev.
+    BRANCH=dev
+    ```
+
+    ** If you are copying/pasting commands, make sure you actually set the right value for VERSION above. **
+
+2.  Make sure you have the latest master.
+
+    ```
+    git checkout master
+    git pull
+    ```
+
+3.  Merge the branch being released into master.
+
+    ```
+    git merge $BRANCH
+    ```
+
+4.  Alter the Unreleased entry in CHANGELOG.md to point to `$VERSION` and
+    update the link at the bottom of the file. Use the format `YYYY-MM-DD` for
+    the year.
+
+    ```diff
+    -## [Unreleased]
+    +## [1.21.0] - 2017-10-23
+    ```
+
+    ```diff
+    -[Unreleased]: https://github.com/uber/prototool/compare/v1.20.1...HEAD
+    +[1.21.0]: https://github.com/uber/prototool/compare/v1.20.1...v1.21.0
+    ```
+
+
+5.  Update the version number in internal/vars/vars.go and verify that it
+    matches what is in the changelog.
+
+    ```
+    sed -i '' -e "s/^const Version =.*/const Version = \"$VERSION\"/" internal/vars/vars.go
+    ```
+
+6.  Create a commit for the release.
+
+    ```
+    git add internal/vars/vars.go CHANGELOG.md
+    git commit -m "Preparing release v$VERSION"
+    ```
+
+7.  Tag and push the release.
+
+    ```
+    git tag -a "v$VERSION" -m "v$VERSION"
+    git push origin master "v$VERSION"
+    ```
+
+8.  Go to <https://travis-ci.org/uber/prototool/builds> and cancel the
```

Comment:
Oof seems hacky, but *shrug*

Reply:
Heh this is basically copy/paste from yarpc-go